abs.yaml

```
@@ -0,0 +1,2 @@
+aggregate_branch: PKG-8364
+aggregate_check: false
```

```suggestion
aggregate_check: false

noarch_upload:
  - osx-64
  - osx-arm64
```
Because of https://github.com/anaconda-distribution/rocket-platform/blob/main/cookbook/flows/config.yaml#L50C1-L51C13